project/GOVERNANCE.md

```diff
@@ -1,17 +1,121 @@
+raise objections and cast their vote. Candidates must be approved by
+at least 66% of the current maintainers by adding their vote on the mailing
+list. Only maintainers of the repository that the candidate is proposed for are
+allowed to vote.
+
+If a candidate is approved, a maintainer will contact the candidate to
+invite the candidate to open a pull request that adds the contributor to
+the MAINTAINERS file. The candidate becomes a maintainer once the pull
+request is merged.
+
+### Removing Maintainers
+
+Maintainers can be removed from the project, either because of changes
+in their own role and/or work or due to inactivity in the project.
+
+#### Stepping Down Policy
+
+Life priorities, interests, and passions can change. If you're a maintainer but
+feel you must remove yourself from the list, inform other maintainers that you
+intend to step down, and if possible, help find someone to pick up your work.
+At the very least, ensure your work can be continued where you left off.
+
+After you've informed other maintainers, create a pull request to remove
+yourself from the MAINTAINERS file.
+
+#### Inactive Maintainer Policy
+
+Similar to the procedure for adding new maintainers, existing maintainers can
+be removed from the list if they do not show significant activity on the
+project. Periodically, the maintainers review the list of maintainers and their
+activity over the last three months.
+
+If a maintainer has shown insufficient activity over this period, a neutral
+person will contact the maintainer to ask if they want to continue being
+a maintainer. If the maintainer decides to step down as a maintainer, they
+open a pull request to be removed from the MAINTAINERS file.
+
+If the maintainer wants to remain a maintainer, but is unable to perform the
+required duties they can be removed with a vote by at least 66% of
+the current maintainers. An e-mail is sent to the
+mailing list, inviting maintainers of the project to vote. The voting period is
+five business days. Issues related to a maintainer's performance should be
+discussed with them among the other maintainers so that they are not surprised
+by a pull request removing them.
+
+## Project Decision Making
+
+Short answer: EVERYTHING IS A PULL REQUEST.
+
+The Moby core engine project is an open-source project with an open design
+philosophy. This means that the repository is the source of truth for EVERY
+aspect of the project, including its philosophy, design, road map, and APIs
+*If it's part of the project, it's in the repo. If it's in the repo, it's part
+of the project.*
+
+As a result, all decisions can be expressed as changes to the
+repository. An implementation change is a change to the source code. An
+API change is a change to the API specification. A philosophy change is
+a change to the philosophy manifesto, and so on.
+
+All decisions affecting moby/moby, big and small, follow the same 3 steps:
+
+* Step 1: Open a pull request. Anyone can do this.
```

Comment:
Great place to use a numbered list. Maybe don't state that it is 3 steps because there may be more later and then you'd have to remember to update that number too. ;)